cli/tools/jupyter/install.rs

```diff
@@ -39,29 +59,73 @@ fn install_icon(
   Ok(())
 }
 
-pub fn install() -> Result<(), AnyError> {
-  let user_data_dir = user_data_dir()?;
-  let kernel_dir = user_data_dir.join("kernels").join("deno");
+pub fn install(
+  maybe_name: Option<&str>,
+  maybe_display_name: Option<&str>,
+  force: bool,
+) -> Result<(), AnyError> {
+  let user_data_dir = if let Ok(env_var) = std::env::var(TEST_ENV_VAR_NAME) {
+    PathBuf::from(env_var)
```

Comment:
```suggestion
  let user_data_dir = if let Some(env_var) = std::env::var_os(TEST_ENV_VAR_NAME) {
    PathBuf::from(env_var)
```

Deja-vu. Should this code be extracted out to a reusable function?